tests/Unit/API/Site/Controllers/MerchantCenter/ContactInformationControllerTest.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace Automattic\WooCommerce\GoogleListingsAndAds\Tests\Unit\API\Site\Controllers\MerchantCenter;
+
+use Automattic\WooCommerce\GoogleListingsAndAds\API\Google\Merchant;
+use Automattic\WooCommerce\GoogleListingsAndAds\API\Google\Settings;
+use Automattic\WooCommerce\GoogleListingsAndAds\API\Site\Controllers\MerchantCenter\ContactInformationController;
+use Automattic\WooCommerce\GoogleListingsAndAds\MerchantCenter\ContactInformation;
+use Automattic\WooCommerce\GoogleListingsAndAds\Options\OptionsInterface;
+use Automattic\WooCommerce\GoogleListingsAndAds\Proxies\RESTServer;
+use Automattic\WooCommerce\GoogleListingsAndAds\Tests\Framework\ContainerAwareUnitTest;
+use Automattic\WooCommerce\GoogleListingsAndAds\Tests\Tools\HelperTrait\MerchantTrait;
+use Automattic\WooCommerce\GoogleListingsAndAds\Utility\AddressUtility;
+use PHPUnit\Framework\MockObject\MockObject;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Class PhoneNumberControllerTest
+ *
+ * @package Automattic\WooCommerce\GoogleListingsAndAds\Tests\Unit\API\Site\Controllers\MerchantCenter
+ *
+ * @since   x.x.x
+ * @property MockObject|ContactInformation $contact_information
+ * @property MockObject|Settings           $google_settings
+ * @property MockObject|Merchant           $merchant
+ * @property MockObject|OptionsInterface   $options
+ * @property MockObject|AddressUtility     $address_utility
+ * @property RESTServer                    $rest_server
+ * @property ContactInformationController  $contact_information_controller
+ */
+class ContactInformationControllerTest extends ContainerAwareUnitTest {
+
+	use MerchantTrait;
+
+	const ROUTE = '/wc/gla/mc/contact-information';
+
+	/**
+	 * Runs before each test is executed.
+	 */
+	public function setUp() {
+		parent::setUp();
+		$this->contact_information            = $this->createMock( ContactInformation::class );
+		$this->google_settings                = $this->createMock( Settings::class );
+		$this->address_utility                = $this->createMock( Settings::class );
```

Comment:
Just as small typo here:
```suggestion
		$this->address_utility                = $this->createMock( AddressUtility::class );
```